src/server.c

```diff
@@ -1871,17 +1871,17 @@ void createSharedObjects(void) {
     shared.noscripterr = createObject(OBJ_STRING,sdsnew(
         "-NOSCRIPT No matching script. Please use EVAL.\r\n"));
     shared.loadingerr = createObject(OBJ_STRING,sdsnew(
-        "-LOADING Redis is loading the dataset in memory\r\n"));
+        "-LOADING The server is loading the dataset in memory\r\n"));
     shared.slowevalerr = createObject(OBJ_STRING,sdsnew(
-        "-BUSY Redis is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
+        "-BUSY The server is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
```

Comment:
This is duplicated here: https://github.com/valkey-io/valkey/pull/205. 

I think this will be hardcoded a bunch of places. I don't want to break compatibility without thinking this through more.

Reply:
I'm also concerned about LOADING error message. I would rather separate those out into their own review and mark it as a major decision.

Reply:
How come?

If -LOADING breaks something, so can the other errors? Maybe all of it should be a major decision?

Clients are supposed to only look at the first uppercased word in the error. (I know not everyone does what they're supposed to do.)

Reply:
@madolson I see [netty](https://github.com/netty/netty/blob/fee3c39d0835ec18d7aace8776db5c5952ce8272/codec-redis/src/main/java/io/netty/handler/codec/redis/FixedRedisMessagePool.java#L44) has some enum values with BUSY, LOADING and MISCONF containing "Redis".

What impact will it have to the client if we change this? It will still probably understand that it's an error, but will it act differently?

Reply:
To me it looks like a string pool, i.e. an optimization. It will accept other messages too but with an extra allocation or so.

This is where it seems to be used: https://github.com/netty/netty/blob/4.1/codec-redis/src/main/java/io/netty/handler/codec/redis/RedisDecoder.java#L264 (read together with the other code linked above)

Reply:
I'm saying people are actually looking for the whole string. I'm not confident that people are just looking for these error messages. I would rather split these out and review them independent, I think these are much more likely to be breaking changes.

Reply:
Yeah they are breaking changes. We shall have them in the next major version IMO.